scipy/stats/setup.py

```diff
@@ -33,6 +42,31 @@ def configuration(parent_package='',top_path=None):
         sources=['_sobol.c', ],
     )
     config.add_data_files('_sobol_direction_numbers.npz')
+    # add BiasedUrn module
```

Comment:
My previous change was supposed remove the spaces from the blank line, but it looks like I removed the line.  Put it back...
```suggestion

    # add BiasedUrn module
```